lib/splitclient-rb/clients/split_client.rb

```diff
@@ -147,10 +147,10 @@ def sanitize_split_names(calling_method, split_names)
         if (split_name.is_a?(String) || split_name.is_a?(Symbol)) && !split_name.empty?
           true
         elsif split_name.is_a?(String) && split_name.empty?
-          @config.logger.warn("#{calling_method}: you passed an empty split_name, split_name must be a non-empty String or a Symbol")
+          @config.logger.warn("#{calling_method}: you passed an empty feature_flag_name, flag name must be a non-empty String or a Symbol")
```

Comment:
update flag_name to feature_flag_name

Reply:
hey @chillaq i am reviewing the spec and my changes are fine. 🤔